contracts/StablecoinConverter.sol

```diff
@@ -261,8 +280,9 @@ contract StablecoinConverter is EpochTokenLocker {
         //                    = (executedBuyAmount * buyTokenPrice / sellTokenPrice) * (feeDenominator - 1) / feeDenominator
         //                    in order to minimize rounding errors, the order is switched
         //                    = (executedBuyAmount * buyTokenPrice / feeDenominator) * (feeDenominator - 1) / sellTokenPrice
-        uint256 sellAmount = (uint256(executedBuyAmount).mul(buyTokenPrice) / (feeDenominator - 1))
-            .mul(feeDenominator) / sellTokenPrice;
+        uint128 scaledFeeDenominator = scale * feeDenominator;
```

Comment:
Funny! We had spoken yesterday about using SafeMath here, but decided it wasn't necessary since this is an internal function that is only ever called by two constant values  (`scale = 1 or 2` and `feeDenominator`). Do you still think we need SafeMath?

Reply:
`feeDenominator` is a constant, but it is a parameter of the constructor.. so I will just use `SafeMath`. Better Safe than sorry!